clickhouse/package-lock.json

```diff
@@ -0,0 +1,12392 @@
+{
+  "name": "@clickhouse-org/click-house",
```

Comment:
you can remove this file and add in the root of the repo, in the file `package.json` `clickhouse` to the workspace list